src/config.json

```diff
@@ -45,6 +45,294 @@
     "satoshilabs.design"
   ],
   "blacklist": [
+    "ag-app-wallet-connect.dev",
+    "ag-wallet-connection.lol",
+    "connectmetawallet.com",
+    "io-walletconnect.com",
+    "mainnetdefiportal.com",
+    "metabridgeapp.com",
+    "modalpulse.com",
+    "org-walletconnect.app",
+    "wallectconnectfix.com",
+    "wallet-connect.xyz",
+    "wallet-connected.xyz",
+    "wallet-connection.com",
+    "wallet-restore.network",
+    "walletcoinect.net",
+    "walletconnct.com",
+    "walletconnect-pay.com",
+    "walletconnect.mobi",
+    "walletconnect.pw",
+    "walletconnectapp.com",
+    "walletconnectlabs.com",
+    "walletconnectpay.com",
+    "walleteconnect.live",
+    "walleteconnect.org",
+    "walletwebsconnect.com",
+    "wallietconnect.site",
+    "wlletconnect.com",
+    "zkfoundatlon.support",
+    "alletconnectw.gitbook.io",
+    "app-walletconnect-help.gitbook.io",
+    "appresolveauth.pages.dev",
+    "atowalletconnect.pages.dev",
+    "awalletconnect.gitbook.io",
+    "balletconnect.gitbook.io",
+    "cloudwalletconne.webflow.io",
+    "cloudwalletconnect.webflow.io",
+    "conecctt-wallet.gitbook.io",
+    "conectwalet.webflow.io",
+    "conectwallet.gitbook.io",
+    "conesectwallet.gitbook.io",
+    "conicthwallets.gitbook.io",
+    "conketwallts.webflow.io",
+    "connect-wllet.gitbook.io",
+    "connectteewallet.gitbook.io",
+    "connectwaallett.gitbook.io",
+    "connetctiwallet.gitbook.io",
+    "contctawallt.webflow.io",
+    "contctwzallt.webflow.io",
+    "cpcontacts.walleconnect.com",
+    "cryptowalletconnect.pages.dev",
+    "cuonnectd-wallte.webflow.io",
+    "dapps-walletconnect.pages.dev",
+    "dappswalletconnect-nwhsa.formstack.com",
+    "dappwalletconnectsync.pages.dev",
+    "ddao-walletconnect.pages.dev",
+    "dose--waluteconnectt.webflow.io",
+    "esnwallet.vercel.app",
+    "example.walletconnect.org",
```

Comment:
This is a legitimate walletconnect.org subdomain.  Are you sure it's malicious?

Reply:
Indeed `walletconnect.org` is legitimate

Reply:
We have unblocked this and shared with WalletConnect a full postmortem